Odata-docs/webapi-8/tutorials/basic-crud.md

```diff
@@ -376,20 +376,14 @@ The following JSON payload shows the expected response:
 ## Request a single entity
 To support this request, we add a controller action named `Get` (or `GetCustomer`) to the `CustomersController` class. The action should accept a single parameter named `key` of type `int` - same type as the entity's key property:
 ```csharp
-public ActionResult<Customer> Get([FromRoute] int key)
```

Comment:
There's no reason for this change. `$expand` works okay even when the return type is `ActionResult<Customer>`.